src/Angular-CSharp.csproj.in

```diff
@@ -7,9 +7,11 @@
     <NoDefaultLaunchSettingsFile Condition="'$(ExcludeLaunchSettings)' == 'True'">True</NoDefaultLaunchSettingsFile>
 <!--#if (!NoSpaFrontEnd) -->
     <SpaRoot>ClientApp\</SpaRoot>
+    <SpaInstallCommand>npm install</SpaInstallCommand>
+    <SpaBuildCommand>npm run build</SpaBuildCommand>
```

Comment:
I've removed the ` -- --configuration production` because of the message:
```
  Option "--prod" is deprecated: No need to use this option as this builder defaults to configuration "production".
```